### PR TITLE
[Metrics File Info] Fix unicode filepath

### PR DESCRIPTION
**Pull request by ybiquitous**
### Summary or Background

Using the `-z` option of `git ls-files`.
See <https://git-scm.com/docs/git-ls-files#Documentation/git-ls-files.txt--z>

This change aims to fix the following error:

```
wc: '"<img onerror=\"alert('\''ok'\'')\" src=\"foo.png\">.js"': No such file or directory
Command [wc -l "<img onerror=\"alert('ok')\" src=\"foo.png\">.js" asdfsider copy 2.yml asdfsider copy.yml asdfsider.yml goodcheck copy 2.yml goodcheck copy.yml goodcheck.yml index.js master-test.copy/first_depth_file.js master-test.copy/master/second_depth_file.js master-test.copy/📁/second_depth_file.js master-test.copy/🗄️.js master-test/first_depth_file.js master-test/master/second_depth_file.js master-test/📁/second_depth_file.js master-test/🗄️.js zzzzz zzzzz;' SHOW TABLES; SELECT '.txt] exited with 1 (Runners::Shell::ExecError)
```

### Issues or Pull Requests

None.

### Checklist

- [x] Add a new entry to [CHANGELOG.md](https://github.com/sider/runners/blob/master/CHANGELOG.md) if this change is notable.

### CODE REVIEW

File: test/smokes/metrics_fileinfo/unicode_filepath/master-test.copy/🗄️.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const test = 'adsf';
```
Comment (dev-yakuza) [inline test/smokes/metrics_fileinfo/unicode_filepath/master-test.copy/🗄️.js:1]: I think this file is not useful for testing for the same reason as above.

File: test/smokes/metrics_fileinfo/unicode_filepath/master-test.copy/master/second_depth_file.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+const test = 'adsf';
```
Comment (dev-yakuza) [inline test/smokes/metrics_fileinfo/unicode_filepath/master-test.copy/master/second_depth_file.js:1]: I think this file is not useful for testing.

The dot(.) in the path is tested.

```
test/smokes/metrics_fileinfo/unicode_filepath/master-test.copy/🗄️.js
```

And the emoji also is tested.


```
test/smokes/metrics_fileinfo/unicode_filepath/master-test/🗄️.js
```

File: test/smokes/metrics_fileinfo/unicode_filepath/master-test/📁/second_depth_file.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+const test = 'adsf';
```
Comment (dev-yakuza) [inline test/smokes/metrics_fileinfo/unicode_filepath/master-test/📁/second_depth_file.js:1]: I think this file is not useful for testing for the same reason as above.

File: test/smokes/metrics_fileinfo/unicode_filepath/goodcheck copy.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+test test test tset
```
Comment (dev-yakuza) [inline test/smokes/metrics_fileinfo/unicode_filepath/goodcheck copy.yml:1]: I think this file is not useful for testing.

The space(` `) in the path is tested.

```
test/smokes/metrics_fileinfo/unicode_filepath/goodcheck copy 2.yml
```

File: test/smokes/metrics_fileinfo/unicode_filepath/index.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+const test = 'adsf';
```
Comment (dev-yakuza) [inline test/smokes/metrics_fileinfo/unicode_filepath/index.js:1]: I think this file is not useful for testing.

The normal path is tested.

```
test/smokes/metrics_fileinfo/unicode_filepath/goodcheck.yml
```

File: test/smokes/metrics_fileinfo/unicode_filepath/master-test/master/second_depth_file.js [hunk 6]
```diff
@@ -0,0 +1,2 @@
+const test = 'adsf';
```
Comment (dev-yakuza) [inline test/smokes/metrics_fileinfo/unicode_filepath/master-test/master/second_depth_file.js:1]: I think this file is not useful for testing.

The normal path with the depth is tested.

```
test/smokes/metrics_fileinfo/unicode_filepath/master-test/first_depth_file.js
```

File: test/smokes/metrics_fileinfo/unicode_filepath/master-test/🗄️.js [hunk 7]
```diff
@@ -0,0 +1,2 @@
+const test = 'adsf';
```
Comment (dev-yakuza) [inline test/smokes/metrics_fileinfo/unicode_filepath/master-test/🗄️.js:1]: I think this file is not useful for testing for the same reason as above.

**Review by dev-yakuza — COMMENTED** [review]
I think there are unuseful test target files. I am not sure the `gitmetadata` folder is needed 🤔 
What do you think?

**Comment by ybiquitous** [comment]
@dev-yakuza Make sense. I've removed useless test files to focus on the bug (e44e8cc4).

**Review by dev-yakuza — APPROVED** [review]
LGTM 👍

**Comment by ybiquitous** [comment]
Thank you for the review! 😄

**Pull request merged**